src/contour/Config.cpp

```diff
@@ -1895,12 +1896,12 @@ std::string YAMLConfigWriter::createString(Config const& c)
     // inside profiles:
     doc.append(fmt::format(fmt::runtime(c.profiles.documentation), fmt::arg("comment", "#")));
     {
-        const auto _ = Offset {};
+        const auto _ = typename Writer::Offset {};
         for (auto&& [name, entry]: c.profiles.value())
         {
             doc.append(fmt::format("    {}: \n", name));
             {
-                const auto _ = Offset {};
+                const auto _ = typename Writer::Offset {};
```

Comment:
is Offset needed to be inside Writer? Maybe we can get rid of the not-so-nice `typename` :)

Reply:
ok, further below I see `Writer::Offset::levels`, I wonder why there it works, but for `Writer::Offset` it doesn't. 🤔

Reply:
we need it here because `Writer` is a template parameter, and name lookup by default looking for class members only, but we need to specify that this is a type